core/include/cubos/core/al/audio_device.hpp

```diff
@@ -51,36 +43,40 @@ namespace cubos::core::al
         /// @brief Forbid copy construction.
         AudioDevice(const AudioDevice&) = delete;
 
-        /// @brief Creates an audio device from a given device @p specifier.
+        /// @brief Creates an audio device.
         /// @see enumerateDevices()
-        /// @param specifier Device specifier (empty for default).
         /// @return Audio device, or nullptr on failure.
-        static std::shared_ptr<AudioDevice> create(const std::string& specifier = "");
-
-        /// @brief Enumerates the available devices.
-        /// @param[out] devices Vector to fill with the available devices.
-        static void enumerateDevices(std::vector<std::string>& devices);
+        static std::shared_ptr<AudioDevice> create();
 
         /// @brief Creates a new audio buffer
+        /// @param filePath File path to create buffer from.
         /// @return Handle of the new buffer.
```

Comment:
This documentation is outdated, but other than that, it is important to specify what is being received here.
What audio format does this function take? Can it receive multiple formats?